### PR TITLE
:beetle: Fix miswrite label

### PR DESCRIPTION
**Pull request by PabloEmidio**

### CODE REVIEW

**Review by CleoMenezesJr — CHANGES_REQUESTED** [review]
Please return the "...". That means it will open another window.

**Comment by PabloEmidio** [comment]
@CleoMenezesJr done

File: src/gtk/window.blp [hunk 1]
```diff
@@ -498,7 +498,7 @@ template EscamboWindow : Adw.ApplicationWindow {
                                     Adw.PreferencesGroup {
                                         Adw.ActionRow create_new_body {
                                             /* icon-name: "list-add-symbolic"; */
-                                            title: _("Create New Cookie...");
+                                            title: _("Create Body...");
```
Comment (CleoMenezesJr) [inline src/gtk/window.blp]: "Create New Body..." is the right way

**Review by CleoMenezesJr — APPROVED** [review]
LGTM

**Pull request merged**